app/views/project/editProject.js

```diff
@@ -57,11 +58,47 @@ let editProject = (cursor) => {
       })
 }
 
+let DeleteProjectDialog = component('DeleteProjectDialog', (cursor) => {
+  
+  let yesCallback = () => {
+    let editCursor = cursor.cursor('editProject')
+    let projectCursor = editCursor.cursor('project')
+    let project = projectCursor.toJS()
+    let qualifiedProjectId = `${project.owner}/${project.projectId}`
+    
+    editCursor.set('isWaiting', true)
+    
+    ajax.delete(`/api/projects/${project.owner}/${project.projectId}`)
+      .then(() => {
+        logger.debug(`Project successfully deleted '${qualifiedProjectId}'`)
+        router.goTo('/')
+      }, (error) => {
+        logger.warn(`Failed to delete project '${qualifiedProjectId}': ${error}`)
+      })
+      .finally(() => {
+        cursor.set('showDeleteProjectDialog', false)
+        editCursor.set('isWaiting', false)
+      })
+  }
+
+  let closeCallback = () => {
+    cursor.set('showDeleteProjectDialog', false)
+  }
+  let title = 'Delete'
+  let message = 'Are you sure you want to delete this project?'
+
+  return notification.question(title, message, yesCallback, closeCallback)
+  
+})
+
 let EditProjectPad = component('EditProjectPad', (cursor) => {
   let editCursor = cursor.cursor('editProject')
   let projectCursor = editCursor.cursor('project')
   let project = projectCursor.toJS()
+  let showDialog = cursor.get('showDeleteProjectDialog') === true
```

Comment:
I think the `=== true` can be dropped. If it has to be a boolean value, rather than truthy, the typical idiom is to use `!!`.